apps/survey/contracts/Survey.sol

```diff
@@ -157,9 +157,10 @@ contract Survey is AragonApp {
             for (uint256 i = 1; i <= previousVote.optionsCastedLength; i++) {
                 OptionCast storage previousOptionCast = previousVote.castedVotes[i];
                 uint256 previousOptionPower = survey.optionPower[previousOptionCast.optionId];
```

Comment:
I don't know how the compiler handles this, but `previousOptionPower` is only used once, and although it's more readable this way maybe a comment would suffice.